storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/core/RocksDbInstance.java

```diff
@@ -215,10 +215,10 @@ public void commit() {
       assertOpen();
       try {
         this.rocksDbTx.commit();
-        close();
       } catch (RocksDBException e) {
-        rollback();
```

Comment:
We're sure there's no way the commit could partially succeed and need to be rolled back?  Should we have a tryRollback method that doesn't throw?

Reply:
Definitely sure - transactions are specifically designed so that they either commit everything or fail to commit and never anything in between.